zxcvbn-cs/README.md

```diff
@@ -101,3 +101,17 @@ The following changes differ from the [original library](https://github.com/mick
 	
 - Change dictionaries load
 	Loads all embedded resources from the directory "zxcvbn.Dictionaries"
+
+- Rename class "Zxcvbn" to "PasswordMetric"
+
+- Rename class "Result" to "PasswordMetricResult"
+
+- Rename translation value "France" to "French"
+
+- Add translation value "Italian"
+
+- Remove not needed namespaces
```

Comment:
This is not an api change

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/95d63aaa67f97ae5e0c0dea5278a314159845cb6